gis/text_to_centreline/sql/mat-view-bylaws_speed_limit_layer.sql

```diff
@@ -278,9 +293,10 @@ UNION
     part_two_without_bylaw.date_added,
     part_two_without_bylaw.date_repealed
    FROM part_two_without_bylaw
+   JOIN gis.centreline cl USING (geo_id, lf_name)
 WITH DATA;
 
-COMMENT ON MATERIALIZED VIEW gis.bylaws_centreline_categorized
+COMMENT ON MATERIALIZED VIEW gis.bylaws_speed_limit_layer
```

Comment:
The start of this comment should confirm that this is the layer one is expecting: Speed limits based on bylaws.